requirements.txt

```diff
@@ -1,4 +1,9 @@
 nest-asyncio>=1.2.0
 qiskit-terra>=0.10
+qiskit-aer
+qiskit-ibmq-provider
 requests>=2.19
 websockets>=7
+pyjwt>=1.7.1
+keyrings.alt>=3.0.0
```

Comment:
Is this really required? Based on the description in the package metadata: https://pypi.org/project/keyrings.alt/ if it is we should find an alternative.

Reply:
The keyrings.alt is convenient in that it is OS agnostic which is good for testing but for any given platform there are preferred keyring implementations.  I will move this from requirements.txt to tox.ini which is more appropriate when building out the test environment.

Reply:
Updated to move keyrings.alt to tox.ini.  This will require that users have a viable keyring implementation when they install.  I can, if necessary, specify platform specific defaults, but most users will likely have a platform specific package already installed.